modules/cco-ccoctl-upgrading.adoc

```diff
@@ -22,18 +22,27 @@ On AWS clusters, some `ccoctl` commands make AWS API calls to create or modify A
 
 .Procedure
 
+. From the directory that contains the installation program, set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
++
+[source,terminal]
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
```

Comment:
This is one way to find the release image pre-update, but it requires the user to download, verify, and run the `openshift-install` binary for their target release.  A more convenient approach is probably to get the pullspec from `oc adm upgrade`, and we have example output for that [over here][1], including:

```
4.9.32  quay.io/openshift-release-dev/ocp-release@sha256:ecdb6d0df547b857eaf0edb5574ddd64ca6d9aff1fa61fd1ac6fb641203bedfa
```

I dunno what the plan is around docs maintenance, but even though that's 4.9 output, it's still the same, or at least very similar to, the output 4.14 `oc` will generate for 4.14 clusters.  However, there are some caveats around mirroring, and I'm noticing now that openshift/oc#1465 seems to have overlooked `oc adm release extract ...`.

https://github.com/openshift/oc/blob/cb4d9dee30e4f1944b179ee653420c45fc8e81dd/pkg/cli/admin/release/info.go#L136-L138

[1]: https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli

Reply:
I'll talk to Sebastian to see if he's got a maintenance-friendly approach :slightly_smiling_face:

Reply:
Ok I think i have accomplished this